backend/app/rest/api/rss.go

```diff
@@ -88,6 +89,54 @@ func (s *Rest) rssSiteCommentsCtrl(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GET /rss/reply?user=userID&site=siteID
+func (s *Rest) rssRepliesCtrl(w http.ResponseWriter, r *http.Request) {
+	userID := r.URL.Query().Get("user")
+	siteID := r.URL.Query().Get("site")
+	log.Printf("[DEBUG] get rss replies to user %s for site %s", userID, siteID)
+
+	data, err := s.Cache.Get(cache.Key(cache.URLKey(r), siteID, userID), func() ([]byte, error) {
+		comments, e := s.DataService.Last(siteID, 100)
+		if e != nil {
+			return nil, e
+		}
+		comments = s.adminService.alterComments(comments, r)
+
+		replies := []store.Comment{}
+		for _, c := range comments {
+			if len(replies) > maxRssItems || c.Timestamp.Add(30*time.Minute).Before(time.Now()) {
+				break
+			}
+			if c.ParentID != "" && !c.Deleted && c.User.ID != userID { // not interested replies to yourself
+				pc, e := s.DataService.Get(c.Locator, c.ParentID)
```

Comment:
this made linter sad "app/rest/api/rss.go:111::warning: declaration of "e" shadows declaration at app/rest/api/rss.go:99 (vetshadow)"